workflow-templates/im-build-db-ci.yml

```
@@ -0,0 +1,186 @@
+name: Build Database CI
+on: 
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+env:
+  DB_NAME: '' # TODO: Fill in your repo's database name
+  DB_SERVER_NAME: 'localhost,1433'
+  DEFAULT_BRANCH: 'main'
+
+  # The following are all for the snapshot step below and can be removed if it's not used
+  SNAPSHOT_PATH: './snapshot' # TODO: Verify this path is correct
+  EXCLUDED_DB_SNAPSHOT_OBJECTS: '' # TODO: Add some comma separated values to this
+
+  # These next variables are for backing up the database. Remove if not needed.
+  BACKUP_PATH: '' # TODO: Update or remove this
+  BACKUP_FILENAME: '' # TODO: Update or remove this
+
+jobs:
+  # TODO: Remove this and it's usages below if no snapshots are created for your database
+  check-for-snapshot-commit:
+    runs-on: ubuntu-latest
+    outputs:
+      should-skip-remaining-workflow-jobs: ${{ steps.check.outputs.skip }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check for snapshot commit
+        id: check
+        shell: pwsh
+        run: |
+          $CurrentBranch = git rev-parse --abbrev-ref HEAD
+          $LastCommitBy = git --no-pager show -s --format="%aN"
+
+          if ($LastCommitBy -eq "github-actions" -and $CurrentBranch -ne "${{ env.DEFAULT_BRANCH }}")
+          {
+            Write-Host "Snapshot was last commit, skipping the other jobs"
+            echo "::set-output name=skip::true"
+          }
+          else
+          {
+            Write-Host "Snapshot was not the last commit, continuing with the other jobs"
+            echo "::set-output name=skip::false"
+          }
+
+  # TODO: Remove this linting job if your project doesn't need/want it
+  lint-migration-files:
+    runs-on: ubuntu-latest
+    needs: check-for-snapshot-commit
+    if: ${{ needs.check-for-snapshot-commit.outputs.should-skip-remaining-workflow-jobs != 'true' }}
```

You did set the output to false above, what do you think about changing it to an affirmative just so there is less mental processing.
```suggestion
    if: ${{ needs.check-for-snapshot-commit.outputs.should-skip-remaining-workflow-jobs == 'false' }}
```